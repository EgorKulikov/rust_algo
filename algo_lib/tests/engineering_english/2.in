he said that that that that that man used
was wrong
