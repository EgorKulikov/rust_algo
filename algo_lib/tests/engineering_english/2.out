he said that . . . . man used
was wrong
